marlowe-cli/src/Language/Marlowe/CLI/Test/Script.hs

```diff
@@ -720,31 +726,48 @@ useTemplate currency = do
         utDepositLovelace
         utWithdrawalLovelace
         timeout'
-    --UseEscrow{..} -> do paymentDeadline' <- toMarloweTimeout paymentDeadline
-    --                    complaintDeadline' <- toMarloweTimeout complaintDeadline
-    --                    disputeDeadline' <- toMarloweTimeout disputeDeadline
-    --                    mediationDeadline' <- toMarloweTimeout mediationDeadline
-    --                    makeContract $ escrow
-    --                       (E.Constant price)
-    --                       seller
-    --                       buyer
-    --                       mediator
-    --                       paymentDeadline'
-    --                       complaintDeadline'
-    --                       disputeDeadline'
-    --                       mediationDeadline'
-    --UseSwap{..} -> do  aTimeout' <- toMarloweTimeout aTimeout
-    --                   bTimeout' <- toMarloweTimeout bTimeout
-    --                   makeContract $ swap
-    --                       aParty
-    --                       aToken
-    --                       (Constant aAmount)
-    --                       aTimeout'
-    --                       bParty
-    --                       bToken
-    --                       (Constant bAmount)
-    --                       bTimeout'
-    --                       Close
+    UseSwap{..} -> do
+      aTimeout' <- toMarloweTimeout utATimeout
+      bTimeout' <- toMarloweTimeout utBTimeout
+      Currency { ccCurrencySymbol=aCurrencySymbol } <- findCurrency utACurrencyNickname
+      Currency { ccCurrencySymbol=bCurrencySymbol } <- findCurrency utBCurrencyNickname
+      let
+        aPartyRef = fromMaybe (WalletRef faucetNickname) utAParty
+        bPartyRef = fromMaybe (WalletRef faucetNickname) utBParty
+      aParty <- buildParty currency aPartyRef
+      bParty <- buildParty currency bPartyRef
+      makeContract $ swap
+          aParty
+          (M.Token aCurrencySymbol utATokenName)
+          (Constant utAAmount)
+          aTimeout'
+          bParty
+          (M.Token bCurrencySymbol utBTokenName)
+          (Constant utBAmount)
+          bTimeout'
+          Close
+    UseEscrow{..} -> do
+      paymentDeadline' <- toMarloweTimeout utPaymentDeadline
+      complaintDeadline' <- toMarloweTimeout utComplaintDeadline
+      disputeDeadline' <- toMarloweTimeout utDisputeDeadline
+      mediationDeadline' <- toMarloweTimeout utMediationDeadline
+      let
```

Comment:
We use here `faucetNickname` as a default for all the parties but the comment below for `UseEscrowe` states something different:

```
     , utSeller            :: Maybe PartyRef   -- ^ Defaults to a wallet with the "Buyer" nickname.
￼    , utBuyer             :: Maybe PartyRef   -- ^ Defaults to a wallet with the "Seller" ncikname.
￼    , utMediator          :: Maybe PartyRef   -- ^ The mediator.